common/djangoapps/util/views.py

```diff
@@ -231,6 +232,8 @@ def _record_feedback_in_zendesk(
         group = zendesk_api.get_group(group_name)
         if group is not None:
             new_ticket['ticket']['group_id'] = group['id']
+    if support_email is not None:
```

Comment:
This seems odd -- if there is no support (recipient) email we still go ahead and attempt to submit the ticket?  Will a ticket be logged in the system even without a recipient email?